assets/css/index.styl

```diff
@@ -35,6 +35,13 @@ h1
 h2
   font-size: 1.5em
   margin: 0.83em 0
+  font-size: 22px;
+  line-height: 22px;
+  letter-spacing: 4px;
+  margin-bottom: 10px;
+  font-weight: 400;
+  text-transform: uppercase;
+  text-align: center;
```

Comment:
Are you sure you want all the h2's centered? That doesn't match the concept you sent me from https://obeygiant.com/fine-art/

![1498508591](https://user-images.githubusercontent.com/1049204/27558642-794e55e4-5a83-11e7-827c-42db69cb61e9.png)